pom.xml

```diff
@@ -94,6 +95,10 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.7.0</version>
+                <configuration>
```

Comment:
Should need to set this as you've set the `<maven.compiler.source>9</maven.compiler.source` and `<maven.compiler.target>9</maven.compiler.target>` properties.  You should refer to the properties if you do want to make this explicit.